Week-6/W6Exercise6.5/Index.js

```diff
@@ -0,0 +1,19 @@
+const findPairWithDifference = (arr, difference) => {
+    const map = new Map();
```

Comment:
Its O(N) in terms of memory and time complexity, can you think of better approach?

Reply:
```
const findPairWithDifference = (arr, difference) => {
  const _arr = [];
  while (arr.length > 0) {
    const first = arr.pop();
    const second = first + difference;
    if (_arr.includes(second)) {
      return 1;
    }
    _arr.push(first);
  }
  return 0;
};
```